.idea/kotlinc.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
```

Comment:
don't add this file. you can git ignore it

Reply:
sure